app/views/courses/_about_schools.html.erb

```diff
@@ -1,6 +1,13 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
   <div data-qa="course__about_schools">
+      <%= render AdviceComponent.new(title: "Where you will train") do %>
+        <% if course.provider_type == "university" %>
```

Comment:
I think this should be
```suggestion
        <% if course.provider_type == "HEI" %>
```

Reply:
So the enum value is 
![image](https://user-images.githubusercontent.com/42515961/107700133-f4acb280-6cae-11eb-9a56-b736feab53c9.png)